### PR TITLE
Add MediaType to BinaryData

### PR DESCRIPTION
**Pull request by JoshLove-msft**
Applies the PR feedback from https://github.com/dotnet/runtime/pull/87744/

### CODE REVIEW

File: src/libraries/System.Memory.Data/src/System/BinaryData.cs [hunk 1]
```diff
@@ -4,6 +4,7 @@
 using System.ComponentModel;
 using System.Diagnostics.CodeAnalysis;
 using System.IO;
+using System.Net.Mime;
```
Comment (JoshLove-msft) [inline src/libraries/System.Memory.Data/src/System/BinaryData.cs:7]: This is only needed for ref doc comments.

Comment (stephentoub) [reply src/libraries/System.Memory.Data/src/System/BinaryData.cs:7]: The using is fine.

**Review by stephentoub — APPROVED** [review]

**Comment by JoshLove-msft** [comment]
Thanks for the review @stephentoub! Can this be merged?

**Comment by stephentoub** [comment]
yup. thanks.

**Pull request merged**